tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md

```diff
@@ -80,15 +105,31 @@ blocks:
   - name: Deploy
     task:
       jobs:
-        - name: 'Everything'
+        - name: Staging
           commands:
             - echo 'Deploying to staging!'
 </code></pre>
 
 Auto-promotions can also be associated to specific branches. Here's how to
```

Comment:
I remember it was hard to find this section, let's add a H3 above: "Continuous deployment on specific branch".

Reply:
Done.